v3/src/components/slider/editable-slider-value.tsx

```diff
@@ -33,25 +36,21 @@ export const EditableSliderValue = ({sliderModel} : IProps) => {
 
   const handleBlur = () => {
     const myFloat = parseFloat(candidate)
-    if (isNaN(myFloat)){
-      setCandidate(sliderModel.globalValue.value.toString())
-    } else {
-      const rounded = myFloat.toFixed(2)
-      const asString = rounded.toString()
+    if (isFinite(myFloat)) {
```

Comment:
`isFinite`🙏

Reply:
I generally find `isFinite()` preferable to `isNaN()` both because infinity should usually be handled as invalid along with `NaN` but also because it doesn't lead to constructs like `!isNaN()` which is a double-negative and seems harder to reason about.